src/ui/pages/Backup/VerifyUser/VerifyUser.tsx

```diff
@@ -0,0 +1,154 @@
+  const { isNarrowView } = useWindowSizeStore();
+
+  const navigate = useNavigate();
+  const handleBackClick = useCallback(() => {
+    navigate(-1);
+  }, [navigate]);
+
+  const { data: user, isLoading } = useQuery({
+    queryKey: ['account/getExistingUser'],
+    queryFn: () => {
+      return accountPublicRPCPort.request('getExistingUser');
+    },
+    useErrorBoundary: true,
+  });
+  const loginMutation = useMutation({
+    mutationFn: ({
+      user,
+      password,
+    }: {
+      user: PublicUser;
+      password: string;
+    }) => {
+      return accountPublicRPCPort.request('login', { user, password });
+    },
+    onSuccess() {
+      zeroizeAfterSubmission();
+      onSuccess();
+    },
+  });
+
+  const inputId = useId();
+
+  if (isLoading) {
+    return null;
+  }
+
+  const faq = (
+    <VStack gap={24} style={{ alignContent: 'start' }}>
+      {isNarrowView ? null : (
+        <div className={styles.faqIcon}>
+          <div style={{ width: 20, height: 20 }}>☝️</div>
+        </div>
+      )}
+      <VStack gap={8}>
+        <UIText kind="small/regular" color="var(--neutral-600)">
+          For security, it's crucial to write down the recovery phrase and store
+          it securely.
+        </UIText>
+        <UIText kind="small/regular" color="var(--neutral-600)">
+          Your recovery phrase is the only way to access your accounts and
+          assets, even if you forget your passcode.
+        </UIText>
+        <UIText kind="small/regular" color="var(--neutral-600)">
+          Never share your recovery phrase or passcode with anyone, including
+          Zerion team members.
+        </UIText>
+      </VStack>
+    </VStack>
+  );
+
+  return (
+    <VStack gap={isNarrowView ? 16 : 56}>
+      <div className={helperStyles.container}>
+        <UnstyledButton
+          onClick={handleBackClick}
+          aria-label="Go Back"
+          className={helperStyles.backButton}
+        >
+          <ArrowLeftIcon style={{ width: 20, height: 20 }} />
+        </UnstyledButton>
+        <Stack
+          gap={isNarrowView ? 0 : 60}
+          direction={isNarrowView ? 'vertical' : 'horizontal'}
+          style={{
+            gridTemplateColumns: isNarrowView ? undefined : '380px auto',
+            justifyContent: 'center',
+          }}
+        >
+          <VStack gap={24} style={{ alignItems: 'start' }}>
+            <VStack gap={8}>
+              <UIText kind="headline/h2">Enter Password</UIText>
+              <UIText kind="body/regular">
+                This password is required to reveal your recovery phrase.
+              </UIText>
+            </VStack>
+            <form
+              onSubmit={(event) => {
+                event.preventDefault();
+                const password = new FormData(event.currentTarget).get(
+                  'password'
+                ) as string | undefined;
+                if (!password) {
+                  return;
+                }
+                if (!user) {
```

Comment:
I think, we can write invariant for user right after `if (isLoading) return null`

Reply:
Basically, this is copied from [here](https://github.com/zeriontech/zerion-wallet-extension/blob/0a6f7715a877100bdd8fe4e088c82522a4a4a144/src/ui/pages/Login/Login.tsx#L82-L91) (or [here](https://github.com/zeriontech/zerion-wallet-extension/blob/0a6f7715a877100bdd8fe4e088c82522a4a4a144/src/ui/components/VerifyUser/VerifyUser.tsx#L71-L81)). I'd like to keep this logic as close as possible to the original so it's easier to refactor identical parts in the future